spring-cloud-starter-stream-sink-task-launcher-local/README.adoc

```diff
@@ -6,12 +6,25 @@ in the message that is received.
 
 Any message that does not contain a `TaskLaunchRequest` will log an error.
 
+== Input
+
+=== Headers
+
+N/A
+
+=== Payload
+
+* `org.springframework.cloud.task.launcher.TaskLaunchRequest`
```

Comment:
Good. Was there a test when this `TaskLaunchRequest` is sent/received as JSON or only serialization variants?